include/wlr/types/wlr_relative_pointer_v1.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_RELATIVE_POINTER_V1_H
+#define WLR_TYPES_WLR_RELATIVE_POINTER_V1_H
+
+#include <wayland-server.h>
+
+
+/**
+ * This protocol specifies a set of interfaces used for making clients able to
+ * receive relative pointer events not obstructed by barriers (such as the
+ * monitor edge or pointer constraints).
+ */
+
+
+/**
+ * A global interface used for getting the relative pointer object for a given
+ * pointer.
+ */
+
+struct wlr_relative_pointer_manager_v1 {
+	struct wl_global *global;
+	struct wl_list resources; // wl_resource_get_link()
+	struct wl_list relative_pointers;
+
+	struct {
+		struct wl_signal destroy;
+		/**
+		 * returns struct wlr_relative_pointer_v1 *relative_pointer
```

Comment:
Hmm, it doesn't really "return" a value. We generally just add `// wlr_relative_pointer_v1` on the same line as the event.